licenses.yaml

```diff
@@ -2807,20 +2806,6 @@ notices:
 
 ---
 
-name: Apache HttpClient
```

Comment:
That's interesting. I'll look into the fact why the license check doesn't complain.

Reply:
Hmm, it doesn't show up in the unchecked licenses.

Reply:
So the version looks consistent:
```
MacBook-Pro-van-Fokko:incubator-druid fokkodriesprong$ mvn dependency:tree | grep -i ":httpclient"
[INFO] |  |  +- org.apache.httpcomponents:httpclient:jar:4.5.9:compile
[INFO] +- org.apache.httpcomponents:httpclient:jar:4.5.9:compile
[INFO] +- org.apache.httpcomponents:httpclient:jar:4.5.9:test
[INFO] |  +- org.apache.httpcomponents:httpclient:jar:4.5.9:compile
[INFO] |  +- org.apache.httpcomponents:httpclient:jar:4.5.9:compile
[INFO] |  +- org.apache.httpcomponents:httpclient:jar:4.5.9:compile
[INFO] |  +- org.apache.httpcomponents:httpclient:jar:4.5.9:provided
[INFO] |  +- org.apache.httpcomponents:httpclient:jar:4.5.9:provided
[INFO] |  +- org.apache.httpcomponents:httpclient:jar:4.5.9:provided
[INFO] |  +- org.apache.httpcomponents:httpclient:jar:4.5.9:provided
[INFO] |  +- org.apache.httpcomponents:httpclient:jar:4.5.9:provided
[INFO] |  +- org.apache.httpcomponents:httpclient:jar:4.5.9:provided
[INFO] |  +- org.apache.httpcomponents:httpclient:jar:4.5.9:provided
[INFO]    +- org.apache.httpcomponents:httpclient:jar:4.5.9:provided
[INFO] |  +- org.apache.httpcomponents:httpclient:jar:4.5.9:compile
[INFO] |  +- org.apache.httpcomponents:httpclient:jar:4.5.9:compile
[INFO] |  |  +- org.apache.httpcomponents:httpclient:jar:4.5.9:provided
[INFO] |  +- org.apache.httpcomponents:httpclient:jar:4.5.9:provided
[INFO] |  +- org.apache.httpcomponents:httpclient:jar:4.5.9:provided
[INFO] |  +- org.apache.httpcomponents:httpclient:jar:4.5.9:provided
[INFO] |  +- org.apache.httpcomponents:httpclient:jar:4.5.9:provided
[INFO] |  +- org.apache.httpcomponents:httpclient:jar:4.5.9:provided
[INFO] |  +- org.apache.httpcomponents:httpclient:jar:4.5.9:provided
[INFO] |     +- org.apache.httpcomponents:httpclient:jar:4.5.9:provided
[INFO] |     +- org.apache.httpcomponents:httpclient:jar:4.5.9:provided
[INFO] |  +- org.apache.httpcomponents:httpclient:jar:4.5.9:provided
[INFO] |  +- org.apache.httpcomponents:httpclient:jar:4.5.9:provided
[INFO] |  |  +- org.apache.httpcomponents:httpclient:jar:4.5.9:provided
[INFO] |  +- org.apache.httpcomponents:httpclient:jar:4.5.9:test
[INFO] |  +- org.apache.httpcomponents:httpclient:jar:4.5.9:test
[INFO]    +- org.apache.httpcomponents:httpclient:jar:4.5.9:test
[INFO] |  +- org.apache.httpcomponents:httpclient:jar:4.5.9:provided
[INFO] |  +- org.apache.httpcomponents:httpclient:jar:4.5.9:test
[INFO]    +- org.apache.httpcomponents:httpclient:jar:4.5.9:test
[INFO] |  |  +- org.apache.httpcomponents:httpclient:jar:4.5.9:provided
[INFO] |  +- org.apache.httpcomponents:httpclient:jar:4.5.9:test
[INFO] |  +- org.apache.httpcomponents:httpclient:jar:4.5.9:provided
[INFO] |  +- org.apache.httpcomponents:httpclient:jar:4.5.9:test
[INFO] |  +- org.apache.httpcomponents:httpclient:jar:4.5.9:provided
[INFO] |  +- org.apache.httpcomponents:httpclient:jar:4.5.9:provided
[INFO] |  +- org.apache.httpcomponents:httpclient:jar:4.5.9:provided
[INFO] |  +- org.apache.httpcomponents:httpclient:jar:4.5.9:provided
[INFO]    +- org.apache.httpcomponents:httpclient:jar:4.5.9:test
[INFO]    +- org.apache.httpcomponents:httpclient:jar:4.5.9:provided
[INFO] |  +- org.apache.httpcomponents:httpclient:jar:4.5.9:provided
[INFO] +- org.apache.httpcomponents:httpclient:jar:4.5.9:compile
[INFO]    |  +- org.apache.httpcomponents:httpclient:jar:4.5.9:compile
```
Also the hadoop one is at 4.5.9:
```
[INFO] +- org.apache.hadoop:hadoop-client:jar:2.8.3:provided
[INFO] |  +- org.apache.hadoop:hadoop-common:jar:2.8.3:provided
[INFO] |  |  +- commons-cli:commons-cli:jar:1.3.1:provided
[INFO] |  |  +- org.apache.commons:commons-math3:jar:3.6.1:provided
[INFO] |  |  +- xmlenc:xmlenc:jar:0.52:provided
[INFO] |  |  +- org.apache.httpcomponents:httpclient:jar:4.5.9:provided
```

Reply:
@clintropolis any further thoughts on this?

Reply:
Hmm, it could be that the license checker doesn't pick it up because nothing directly references this which just gets packed into the hadoop client folder at distribution packaging time.

I pulled your branch, and definitely still see version 4.5.2 in there though:
```
$ ls -1 | grep -C2 httpclient
hadoop-yarn-server-common-2.8.3.jar
htrace-core4-4.0.1-incubating.jar
httpclient-4.5.2.jar
httpcore-4.4.4.jar
jackson-core-asl-1.9.13.jar
```
which is what this entry in the `licenses.yaml` file is for, so I still think this entry should be added back.

Reply:
Interesting, I've revoked the change of adding the license for now.